### DISCUSSION TITLE
I hope to see the saved passwords

### DISCUSSION THREAD

**Discussion by crazyweeds** [Eugeny/tabby#5967] [General]
Can you add an option？

**Comment by Eugeny** [comment]
The SSH client already offers to save passwords.

**Reply by crazyweeds** [reply]
like this.

![image](https://user-images.githubusercontent.com/16688520/159223628-7773bdd1-78b5-4d52-b989-9b90928e92e3.png)

**Reply by Eugeny** [reply]
When you connect for the first time, you'll be automatically offered to save the password to your OS keychain.

**Reply by crazyweeds** [reply]
Screenshot from:
https://www.nuoshell.com/

**Comment by RalphBouRaad** [comment]
Do you happen to have any updates on the above?

I work on many production servers, each with its own password. 
I know I saved the passwords with the profile, but seeing it would be so helpful, especially when I want to use sudo commands and am prompted to re-enter my password.

**Comment by ianaflous** [comment]
Hello, if you have saved your passwords in tabby vault, you can show them like this :
<img width="951" height="171" alt="image" src="https://github.com/user-attachments/assets/3caefea1-2cf8-4c62-a0d8-a57f28accf84" />

**Reply by crazyweeds** [reply]
Thanks, but I have given up on tabby and switched to something else

**Reply by vcth4nh** [reply]
May I ask, what did you use?
